### DISCUSSION TITLE
重命名下载文件

### DISCUSSION THREAD

**Discussion by alephpi** [yutto-dev/yutto#381] [Q&A]
你好，我希望对yutto下载的音频文件做ASR（不是每个视频都有字幕 :cry: ），但是每次下载的文件名称都不一样，请问yutto是否支持重命名音频文件？这样能更方便的整合到我的pipeline里。

**Comment by SigureMo** [comment]
参考 https://github.com/yutto-dev/yutto#%E5%AD%98%E6%94%BE%E5%AD%90%E8%B7%AF%E5%BE%84%E6%A8%A1%E6%9D%BF
